components/login.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <title>Shoemall | Login</title>
+</head>
+<body>
+
+    <nav class="nav">
+        <img class="nav__logo" src="./images/logo.png" alt="logo-img"/>
+        <div class="nav__option">
+            <div class="nav__short">
+                <i class="fas fa-bars"></i>
+            </div>
+            <ul>
+                <li><p><i class="far fa-heart"></i> Wishlist</p></li>
+                <li><p><i class="fas fa-shopping-cart"></i> Cart</p>
+                </li>
+            </ul>
+        </div>
+    </nav>     
+
+
+    <div class="login__outer">
+        <form class="login">
+           <h1 class="login__title">Log in</h1>
+           <div class="login__fields">
+              <p for="email">Email</p>
```

Comment:
Use `label` for inputs instead of `p`

Reply:
I wanted to give a name which is "Email" on top of the input tag but as the label is an inline element it is not working properly for me because i need to add an extra <br> tag over there so to avoid that i switch to <p> tag.